[Merged interops] Warnings on build

The current build contains a version of `Microsoft.VisualStudio.ProjectSystem.Query` that causes the following build warning:
```
CSC : warning CS1762: A reference was created to embedded interop assembly 'Microsoft.VisualStudio.Imaging.Interop.14.0.DesignTime, Version=14.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a' because of an indirect reference to that assembly created by assembly 'Microsoft.VisualStudio.Pr
ojectSystem.Query, Version=17.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a'. Consider changing the 'Embed Interop Types' property on either assembly. [G:\project-system\src\Microsoft.VisualStudio.ProjectSystem.Managed.VS\Microsoft.VisualStudio.ProjectSystem.Managed.VS.csproj]
```

The solution is for CPS team to update this binary and then we'll be able to re-add `/warnaserror` into our build.cmd

--- a/build.cmd
+++ b/build.cmd
@@ -57,8 +57,7 @@ if "%OptDiagnostic%" == "true" (
 
 call "%Root%\build\script\SetVSEnvironment.cmd" || exit /b 1
 
-REM TODO: Merged PIAs /warnaserror needs to be added again
-msbuild %Root%build\proj\Build.proj /m /nologo /clp:Summary /nodeReuse:%OptNodeReuse% /p:Configuration=%BuildConfiguration% /p:Build=%OptBuild% /p:Rebuild=%OptRebuild% /p:Deploy=%OptDeploy% /p:Test=%OptTest% /p:IntegrationTest=%OptIntegrationTest% /p:Sign=%OptSign% /p:CIBuild=%OptCI% /p:EnableIbc=%OptIbc% /p:ClearNuGetCache=%OptClearNuGetCache% %LogCmdLine% %RootSuffixCmdLine%
+msbuild %Root%build\proj\Build.proj /m /nologo /warnaserror /clp:Summary /nodeReuse:%OptNodeReuse% /p:Configuration=%BuildConfiguration% /p:Build=%OptBuild% /p:Rebuild=%OptRebuild% /p:Deploy=%OptDeploy% /p:Test=%OptTest% /p:IntegrationTest=%OptIntegrationTest% /p:Sign=%OptSign% /p:CIBuild=%OptCI% /p:EnableIbc=%OptIbc% /p:ClearNuGetCache=%OptClearNuGetCache% %LogCmdLine% %RootSuffixCmdLine%
 set MSBuildErrorLevel=%ERRORLEVEL%
 
 if "%OptDiagnostic%" == "true" if "%OptCI%" == "false" (

--- a/build/import/HostAgnostic.props
+++ b/build/import/HostAgnostic.props
@@ -94,7 +94,6 @@
            or '%(FileName)' == 'Microsoft.VisualStudio.Shell.Interop.15.3.DesignTime'
            or '%(FileName)' == 'Microsoft.VisualStudio.Shell.Interop.15.7.DesignTime'
            or '%(FileName)' == 'Microsoft.VisualStudio.Shell.Interop.15.8.DesignTime'
-           or '%(Filename)' == 'Microsoft.VisualStudio.Imaging.Interop.14.0.DesignTime'
            or '%(FileName)' == 'Microsoft.VisualStudio.TextManager.Interop.12.1.DesignTime'
            or '%(FileName)' == 'Microsoft.VisualStudio.ProjectSystem.Interop'
            or '%(FileName)' == 'stdole'
@@ -115,6 +114,7 @@
            or '%(FileName)' == 'EnvDTE80'
            or '%(FileName)' == 'EnvDTE90'
            or '%(FileName)' == 'EnvDTE100'
+           or '%(Filename)' == 'Microsoft.VisualStudio.Imaging.Interop.14.0.DesignTime'
            or '%(FileName)' == 'Microsoft.Internal.VisualStudio.Shell.Interop.14.1.DesignTime'
            or '%(FileName)' == 'Microsoft.Internal.VisualStudio.Shell.Interop.14.2.DesignTime'">
         <EmbedInteropTypes>false</EmbedInteropTypes>

--- a/build/import/Packages.targets
+++ b/build/import/Packages.targets
@@ -65,7 +65,7 @@
     <PackageReference Update="Microsoft.VisualStudio.Threading"                                       Version="16.10.53-alpha" />
     <PackageReference Update="Microsoft.VisualStudio.Threading.Analyzers"                             Version="16.10.53-alpha" />
     <PackageReference Update="Microsoft.VisualStudio.Utilities"                                       Version="17.0.0-preview-2-31223-026" />
-    <PackageReference Update="Microsoft.VisualStudio.Validation"                                      Version="16.9.32" />
+    <PackageReference Update="Microsoft.VisualStudio.Validation"                                      Version="17.0.1-alpha" />
     <PackageReference Update="Microsoft.VisualStudio.Workspace.VSIntegration"                         Version="16.7.47-preview-0001" />
     <PackageReference Update="Microsoft.VisualStudio.XmlEditor"                                       Version="17.0.0-preview-2-31223-026" />
     <PackageReference Update="Microsoft.VSDesigner"                                                   Version="17.0.0-preview-2-31223-026" />
@@ -75,9 +75,9 @@
     
 
     <!-- CPS -->
-    <PackageReference Update="Microsoft.VisualStudio.ProjectSystem"                                   Version="17.0.77-pre-g62a6cb5699" />
+    <PackageReference Update="Microsoft.VisualStudio.ProjectSystem"                                   Version="17.0.89-pre-geec7dc0def" />
     <PackageReference Update="Microsoft.VisualStudio.ProjectSystem.Analyzers"                         Version="16.10.127-pre" />
-    <PackageReference Update="Microsoft.VisualStudio.ProjectSystem.Query"                             Version="17.0.77-pre-g62a6cb5699" />
+    <PackageReference Update="Microsoft.VisualStudio.ProjectSystem.Query"                             Version="17.0.89-pre-geec7dc0def" />
     <PackageReference Update="Microsoft.VisualStudio.ProjectSystem.SDK.Tools"                         Version="16.10.127-pre" />
     
     <!-- Roslyn -->

--- a/build/import/Versions.props
+++ b/build/import/Versions.props
@@ -3,7 +3,7 @@
   <PropertyGroup>
     <!-- IMPORTANT: These assembly versions must always be 3 digits -->
     <MinimumRequiredVSVersion>16.8.1</MinimumRequiredVSVersion> <!-- We should try to keep this up to date with the highest version available on build machines -->
-    <ProjectSystemVersion>17.0.1</ProjectSystemVersion> <!-- This should always match with the VS release we're targeting -->
+    <ProjectSystemVersion>17.0.0</ProjectSystemVersion> <!-- This should always match with the VS release we're targeting -->
     <VersionBase>$(ProjectSystemVersion)</VersionBase>
     <PreReleaseVersionLabel>beta1</PreReleaseVersionLabel>
     <SuppressNETCoreSdkPreviewMessage>true</SuppressNETCoreSdkPreviewMessage>

--- a/src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/ProjectSystem/VS/Query/PropertyPages/UIPropertyDataProducer.cs
+++ b/src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/ProjectSystem/VS/Query/PropertyPages/UIPropertyDataProducer.cs
@@ -58,11 +58,10 @@ namespace Microsoft.VisualStudio.ProjectSystem.VS.Query
                 newUIProperty.ConfigurationIndependent = !property.IsConfigurationDependent();
             }
 
-            //TODO: Merged PIAs - this property needs to be merged in by CPS to dev17 packages 
-            //if (requestedProperties.IsReadOnly)
-            //{
-            //    newUIProperty.IsReadOnly = property.ReadOnly;
-            //}
+            if (requestedProperties.IsReadOnly)
+            {
+                newUIProperty.IsReadOnly = property.ReadOnly;
+            }
 
             if (requestedProperties.HelpUrl)
             {

